### PR TITLE
Upgrade Quantil to Platinum sponsor

### LINKED ISSUE #386: Upgrade Quantil to Platinum sponsor
And remove statuspage from sponsors

### PR DESCRIPTION
**Pull request by xbpcb**
Fix #386

### CODE REVIEW

**Comment by MartinKolarik** [comment]
This should probably include the homepage and footer update as well.

**Comment by jimaek** [comment]
@xbpcb yeah lets add them to the homepage header and remove from footer

File: src/views/components/landing-top.html [hunk 1]
```diff
@@ -217,6 +217,10 @@ <h2>fast, reliable, and automated</h2>
 				<div class="sponsor">
 					<a target="_blank" rel="noopener" href="https://bunny.net"><img src="{{@shared.assetsHost}}/img/landing/bunny.png" srcset="{{@shared.assetsHost}}/img/landing/bunny@2x.png 2x" title="Sponsored by bunny.net"></a>
 				</div>
+
+				<div class="sponsor">
+					<a target="_blank" rel="noopener" href="https://www.quantil.com"><img src="{{@shared.assetsHost}}/img/landing/quantil.png" srcset="{{@shared.assetsHost}}/img/landing/quantil@2x.png 2x" title="Sponsored by Quantil"></a>
```
Comment (MartinKolarik) [inline src/views/components/landing-top.html:222]: The png here needs to be resized to 50px in height (keep whitespace, just resize the whole thing).

**Comment by MartinKolarik** [comment]
Looking at the footer, it seems we should also add cloudns there?

**Review by MartinKolarik — APPROVED** [review]

**Pull request merged**